Modules/Loadable/Data/qSlicerDataModule.cxx

```diff
@@ -91,7 +91,7 @@ void qSlicerDataModule::setup()
   this->Superclass::setup();
 
   vtkSlicerCamerasModuleLogic* camerasLogic =
-    vtkSlicerCamerasModuleLogic::SafeDownCast(this->moduleLogic("Cameras"));
+    vtkSlicerCamerasModuleLogic::SafeDownCast(this->moduleLogic(tr("Cameras")));
```

Comment:
By the way, is really this string a translatable one ?

Reply:
I also have the same doubts when it's about `On` and `Off` strings present on `Modules/Loadable/Data/Logic/vtkSlicerDataModuleLogic.cxx:58`. 

```c++
void vtkSlicerDataModuleLogic::PrintSelf(ostream& os, vtkIndent indent)
{
  this->Superclass::PrintSelf(os, indent);

  os << indent << "AutoRemoveDisplayAndStorageNode: " <<
    (AutoRemoveDisplayAndStorageNodes ? "On" : "Off") << "\n";

 }
```

Reply:
> By the way, is really this string a translatable one ?

It is the module name, so it should not be translated. I just first wanted to find a way how to mark these non-obvious cases in the code to ensure that they don't get translated anywhere.

> doubts when it's about On and Off strings present on

We only translate interface for users. We keep developer communication, such as logs, debugging outputs (such as PrintSelf), non-user-provided file contents (such as fields names and tokens in MRML scene files) the same, regardless of the current language.

Please add all these information (don't translate module name, file content, file extension, developer communication, such as log messages, PrintSelf output) to the [developer manual] (https://github.com/Slicer/SlicerLanguagePacks/blob/main/DevelopersManual.md).

Reply:
Got it, thanks ! 

The right updates will be done on the developer manual.

For modules, there are some folders that contains codes which should not have translatable strings. We can thereby exclude them from the marking process. It's currently the case of `testing` folders. Can I do the same with modules' `logic` folders without risking to miss out any translatable string ?

I think, as a side note, I can add the list of those folders to the developer manual too so that to precise that they should not contain translatable strings.

Reply:
> For modules, there are some folders that contains codes which should not have translatable strings. We can thereby exclude them from the marking process. It's currently the case of testing folders.

Yes, you can exclude testing folders. Please add to the developer manual the lupdate command-line arguments that you use to extract/update the strings.

> Can I do the same with modules' logic folders without risking to miss out any translatable string ?

Translation of strings coming from logic classes is a complex topic. First we need to make the decision if we allow logic classes to generate any user-visible strings and if yes then where they need to be translated (in logic classes or in GUI classes). You can ignore Logic, MRML, MRMLDM, VTKWidgets folders from your analysis, and I've added an issue (https://github.com/Slicer/Slicer/issues/6647) to remember to deal with them later.

Reply:
Alright, that's good then. Thanks !